keps/sig-instrumentation/1933-secret-logging-static-analysis/kep.yaml

```diff
@@ -18,7 +18,7 @@ approvers:
 - "@dashpole"
 - "@ehashman"
 #prr-approvers:
-#- TBD
```

Comment:
You need to update the stage and latest milestone in this file, and that will prompt you to add a file for the PRR review.

Reply:
Done.